databases/week2/databases-week2-Homework/more_queries.sql

```diff
@@ -0,0 +1,50 @@
+-- Get all tasks assigned to users whose email ends in @spotify.com
+SELECT t.title
+FROM user_task ut 
+INNER JOIN task t ON ut.task_id=t.id 
+INNER JOIN (
+    SELECT u.id, u.email
+    FROM user u
+    WHERE email LIKE '%@spotify.com'
+) u ON ut.user_id=u.id;
+
+
+-- Get all taks for 'Donald Duck' where status 'not started' 
+SELECT t.title
+FROM task t
+INNER JOIN user_task ut ON t.id = ut.task_id
+INNER JOIN status s ON t.status_id = s.id 
+WHERE ut.user_id = (
+    SELECT id 
+    FROM user
+    WHERE name LIKE '%Donald Duck%'
```

Comment:
What would happen if there are more than one user with the name LIKE Donald Duck?

Reply:
I think If they pass the condition status.name = not started then they output as well if not, nothing will happen.